src/tsickle.ts

```diff
@@ -634,6 +634,11 @@ class Annotator extends ClosureRewriter {
    *     emit it as is and visit its children.
    */
   maybeProcess(node: ts.Node): boolean {
+    if (this.file.statements.indexOf(node as ts.Statement) !== -1) {
+      const pos = this.file.getLineAndCharacterOfPosition(node.getFullStart() + 1);
+      this.sourceMapper.addMapping(
+            node, {line: pos.line, column: pos.character, position: node.getFullStart() + 1}, {line: this.position.line + 1, column: 0, position: this.position.position + 1}, 1);
```

Comment:
this repeats a bunch of logic regarding indices that seems to belong into the parent class where we handle the source mapping stuff. You could introduce a method there, then you also wouldn't need to open up the visibility.